common.blocks/video/video.css

```diff
@@ -41,3 +41,23 @@
     font-weight: bold;
     color: #fff;
 }
+
+@media screen and (max-width: 773px) {
+    .video,
+    .video_playing,
+    .video__player {
+        position: absolute;
+        left: 0;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        margin: 0;
+        width: 100%;
```

Comment:
with `top, left, right, bottom` is there real reason to have `width, height and box-sizing`?